### ISSUE TITLE
Newbie setup issue

### ISSUE THREAD

**Issue by paseman** [SpotlightKid/python-rtmidi#139]
I am unfamiliar with MIDI in general.
I am on a Mac
In "Audio MIDI Setup", I double clicked IAC Driver, checked "Device is online" and the "ports list" list Bus1.
I run the usage example: https://pypi.org/project/python-rtmidi/
Silence.

Running this:
import time
import rtmidi
print(rtmidi.get_compiled_api())
midiout = rtmidi.MidiOut()
print("midiout:",midiout)
available_ports = midiout.get_ports()
print("available_ports",available_ports)

gives:

[1]
available_ports ['IAC Driver Bus 1']
midiout.open_port(0) <rtmidi._rtmidi.MidiOut object at 0x7fb1a8380590>

The damning part is that the package that uses rtmidi worked just fine yesterday.
But then I tried downloading Macports (which required several reboots) and it no longer functions.
Thanks

**Comment by SpotlightKid** [comment]
Sorry, I am not sure what your problem is exactly? Can you rephrase it? And please show every code and its invocation and include its full output.

**Comment by SpotlightKid** [comment]
Closing this due to no further feedback or clarification.

Feel free to re-open if the problem persists and you have additional information.
